srcpkgs/git-interactive-rebase-tool/template

```diff
@@ -0,0 +1,12 @@
+# Template file for 'git-interactive-rebase-tool'
+pkgname=git-interactive-rebase-tool
+version=2.1.0
+revision=1
+archs="i686 x86_64"
+build_style=cargo
+short_desc="Terminal based sequence editor for git interactive rebase"
```

Comment:
I really doubt the usefulness of this tool. It's a TUI tool, in which way it improves vim/emacs?
(Let's discount the cargo cult of rustify all the things).

Reply:
It's quite useful. You should try it :)

`git-interactive-rebase-tool` paired with `tig` makes for a powerful alternative to magit, for example.

I primarily use [vis](https://sr.ht/~martanne/vis/) as my preferred editor and having powerful external tools like these help with the convenience that similar tools in the emacs ecosystem or vim plugin ecosystem provide.

vis is an editor that subscribes to a [unix as ide](https://blog.sanctum.geek.nz/series/unix-as-ide/) philosophy and tries to primarily leverage external tools instead of trying to write those as plugins in the editor's extension language.